rules/implementation.js

```diff
@@ -0,0 +1,66 @@
+const {
+  is
+} = require('bpmnlint-utils');
+
+const implementationAttributes = [
+  'camunda:expression',
+  'camunda:delegateExpression',
+  'camunda:class',
+  'camunda:type'
+];
+
+/**
+ * Rule that reports the usage of collapsed sub-processes.
+ */
+module.exports = function() {
+
+  function check(node, reporter) {
+    if (is(node, 'camunda:ServiceTaskLike')) {
+
+      // quirk: bpmn:MessageEventDefinition is a camunda:ServiceTaskLike
+      // even though it's not a bpmn:Task
+      if (is(node, 'bpmn:MessageEventDefinition')) {
+        return;
+      }
+
+      if (
+        hasConnector(node) ||
+        hasAnyAttribute(node, implementationAttributes)
+      ) {
+        return;
+      }
+
+      if (is(node, 'bpmn:BusinessRuleTask') && hasAttribute(node, 'camunda:decisionRef')) {
+        return;
+      }
+
+      reporter.report(node.id, 'Implementation should be specified');
```

Comment:
I'd rather use `XY is missing here` to better align with our existing wording.

Reply:
`Implementation is missing`